facturacion_mexico/facturacion_fiscal/tests/test_nota_credito_descuento_relacion_01.py

```
@@ -0,0 +1,530 @@
+				return_value=True,
+			),
+		):
+			inst._detect_nota_credito_motivo()
+		self.assertEqual(inst.fm_tipo_nota_credito, "Devolución de mercancía")
+
+
+class TestGuardTimbradoContrato(FrappeTestCase):
+	"""Contrato del guard pre-timbrado (Opción X, punto 9): NC marcada Descuento debe estar
+	contabilizada contra la cuenta configurada; si no, se bloquea."""
+
+	def _bloquea(self, ffm, si, cuenta):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_bloquea`

(ANN202)